src/indicator/renderer/relativeStrengthIndex.js

```diff
@@ -19,9 +19,17 @@ export default function() {
 
     var rsi = function(selection) {
 
-        multiSeries.xScale(xScale)
+        multi
+            .xScale(xScale)
             .yScale(yScale)
-            .series([rsiLine, annotations])
+            .series([annotations, rsiLine])
+            .decorate(function(g, data, index) {
```

Comment:
any reason why you've changed the order from `series().mapping().decorate()` to `series().decorate().mapping()`?